doc/changes/v3.0.0.rst

```diff
@@ -0,0 +1,356 @@
+  the ``approx`` (:py:class:`~xgboost.DMatrix`) method. (:pr:`10529`, :pr:`10980`, :pr:`10342`)
+- New demos and documents for external memory, including distributed training. (:pr:`11234`,
+  :pr:`10929`, :pr:`10916`, :pr:`10426`, :pr:`11113`)
+- Reduced binary cache size and memory allocation overhead by not writing the cut matrix. (:pr:`10444`)
+- More feature coverage, including categorical data and all objective functions, including
+  quantile regression. In addition, various prediction types like SHAP values are
+  supported. (:pr:`10918`, :pr:`10820`, :pr:`10751`, :pr:`10724`)
+
+Significant updates for the GPU-based external memory training implementation. (:pr:`10924`,
+:pr:`10895`, :pr:`10766`, :pr:`10544`, :pr:`10677`, :pr:`10615`, :pr:`10927`, :pr:`10608`, :pr:`10711`)
+
+- GPU-based external memory supports both batch-based and sampling-based training. Before
+  the 3.0 release, XGBoost concatenates the data during training and stores the cache on
+  disk. In 3.0, XGBoost can now stage the data on the host and fetch them by
+  batch. (:pr:`10602`, :pr:`10595`, :pr:`10606`, :pr:`10549`, :pr:`10488`, :pr:`10766`, :pr:`10765`, :pr:`10764`, :pr:`10760`, :pr:`10753`,
+  :pr:`10734`, :pr:`10691`, :pr:`10713`, :pr:`10826`, :pr:`10811`, :pr:`10810`, :pr:`10736`, :pr:`10538`)
+- XGBoost can now utilize `NVLink-C2C` for GPU-based external memory training and can
+  handle up to terabytes of data.
+- Support prediction cache (:pr:`10707`).
+- Automatic page concatenation for improved GPU utilization (:pr:`10887`).
+- Improved quantile sketching algorithm for batch-based inputs. See section for new
+  :ref:`features <3_0_features>` for more info.
+- Optimization for nearly-dense input, see the section for :ref:`optimization
+  <3_0_optimization>` for more info.
+
+See our latest document for details :doc:`/tutorials/external_memory`. The PyPI package
+(``pip install``) doesn't have ``RMM`` support, which is required by the GPU external
+memory implementation. To experiment, you can compile XGBoost from source or wait for the
+conda package to be available.
+
+.. _3_0_networking:
+
+**********
+Networking
+**********
+
+Continuing the work from the previous release, we updated the network module to improve
+reliability. (:pr:`10453`, :pr:`10756`, :pr:`11111`, :pr:`10914`, :pr:`10828`, :pr:`10735`, :pr:`10693`, :pr:`10676`, :pr:`10349`,
+:pr:`10397`, :pr:`10566`, :pr:`10526`, :pr:`10349`)
+
+The timeout option is now supported for NCCL using the NCCL asynchronous mode (:pr:`10850`,
+:pr:`10934`, :pr:`10945`, :pr:`10930`).
+
+In addition, a new :py:class:`~xgboost.collective.Config` class is added for users to
+specify various options including timeout, tracker port, etc for distributed
+training. Both the Dask interface and the PySpark interface support the new
+configuration. (:pr:`11003`, :pr:`10281`, :pr:`10983`, :pr:`10973`)
+
+****
+SYCL
+****
+
+Continuing the work on the SYCL integration, there are significant improvements in the
```

Comment:
Starting from this release, sycl-plugin is fully launchable. I guess we should mention it in the release notes.

Reply:
Not entirely sure what fully launchable means, I will say "the SYCL-plugin is approaching feature-complete, users can start working on SYCL devices for in-core training and inference", what do you think?

Reply:
sounds nice for me